### PR TITLE
merging main into production

### PR DESCRIPTION
**Pull request by skininja673**
PLEASE REVIEW MY WORK!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
```
Comment (wendiz16) [inline src/App.js:1]: maybe just removing it if you don't need it?

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+                <Navbar />
+                <Routes>
+                    <Route path="/" element={<Home />} />
+                    <Route path="about" element={<About />} />
+                    <Route path="cocktail/:id" element={<SingleCocktail />} />
+                    <Route path="*" element={<Error />} />
+                </Routes>
+            </Router>
+        </div>
+    );
```
Comment (wendiz16) [inline src/App.js:27]: I like your code here! a nice structure for react router!

File: src/components/Cocktail.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Cocktail = ({ image, name, id, info, glass }) => {
+    return (
+        <section className="cocktail">
```
Comment (wendiz16) [inline src/components/Cocktail.js:6]: maybe adding a wrapper for the section?

File: src/App.js (hunk 2 above)
Comment (wendiz16) [inline src/App.js:27]: maybe adding a footer here?

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,28 @@
...
-  );
+    return (
+        <div>
+            <Router>
+                <Navbar />
+                <Routes>
+                    <Route path="/" element={<Home />} />
+                    <Route path="about" element={<About />} />
+                    <Route path="cocktail/:id" element={<SingleCocktail />} />
+                    <Route path="*" element={<Error />} />
```
Comment (wendiz16) [inline src/App.js:22]: do you really need this?

File: src/components/Loading.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Loading = () => {
```
Comment (wendiz16) [inline src/components/Loading.js:3]: do you see this animation loading?

Comment (wendiz16) [reply src/components/Loading.js:3]: it is just a plus..

File: src/index.css [hunk 6]
```diff
@@ -1,13 +1,599 @@
...
+}
+
+/* end of setup */
+
+/* VARIABLES */
+:root {
+    --primaryColor: #cd7300;
+    --mainWhite: #fff;
+    --mainBackground: #f1f5f8;
+}
```
Comment (wendiz16) [inline src/index.css:234]: please make a variable for box shadow and reduce the redundant styling.

Comment (skininja673) [reply src/index.css:234]: Nice catch. Its done ! 🚀

File: src/components/Loading.js (hunk 5 above)
Comment (skininja673) [reply src/components/Loading.js:3]: No I dont see animation loading yet. Probably its my stretch task which I will do later.

File: src/App.js (hunk 2 above)
Comment (skininja673) [reply src/App.js:27]: Thanks for pointing this out, its implemented now. Thanks

File: src/App.js (hunk 4 above)
Comment (skininja673) [reply src/App.js:22]: Its a stretch task to implement Link for this error to navigate user to this component.

**Pull request merged**